### DISCUSSION TITLE
How to change the length of the separation lines in eldoc?

### DISCUSSION THREAD

**Discussion by linwaytin** [joaotavora/eglot#1157] [General]
I prefer to have the information shown near the cursor, when it hovers on a symbol. I use `eldoc-doc` for this purpose, and I get [this](https://i.imgur.com/8O0iEIJ.png). 

As seen in the screenshot, the line is too long for the child frame and gets wrapped. How can I change the length of the line?

In fact, I think the empty line and the separation line are somewhat redundant and occupy space. It would be nice if the user can change them.

Thank you.

**Comment by joaotavora** [comment]
This is in a large measure eldoc specific question and should be posed to Emacs's bug tracker, so that more Emacs people can be involved. Send mail to `bug-gnu-emacs@gnu.org`.  Eldoc/Eglot integration has a bit to go still.

**Reply by linwaytin** [reply]
Ok...so the format is not controlled by `eglot`?

**Reply by joaotavora** [reply]
It's controlled by a number of things: the server, Eglot, ElDoc, markdown-mode. I think it's good to discuss this issue closer to the context of where Eglot *and* ElDoc are maintained, and the Emacs bug tracker is a good place to do that.

**Reply by linwaytin** [reply]
Got you. Have sent an email to the Emacs bug tracker.

**Reply by joaotavora** [reply]
Once you get the confirmation from the bug tracker, please CC me in a reply so that I can know the bug number and it makes it into my inbox. next time you can also just add the `CC:` or `XDebbugs-CC:` in the initial report.  My email address can be easily found in the eglot.el source file.
